src/main/java/seedu/storemando/logic/commands/EditCommand.java

```diff
@@ -52,6 +52,8 @@
     public static final String MESSAGE_DUPLICATE_ITEM = "This item already exists in storemando.";
     public static final String MESSAGE_NO_CHANGE = "Item not edited! Specified change in item details same as "
         + "original.";
+    public static final String MESSAGE_ITEM_EXPIRED_WARNING = "\nWarning: Item has already expired!";
```

Comment:
Maybe warning can be in all caps?

Reply:
I think we can leave display it in red color in our GUI instead of using all caps to warn. I feel it would be friendlier that way.